test/commons/color/get-background-color.js

```diff
@@ -190,6 +190,64 @@ describe('color.getBackgroundColor', function () {
 		assert.deepEqual(bgNodes, [target]);
 	});
 
+	it('should count a tr as a background element', function () {
+		fixture.innerHTML = '<div style="background-color: #007acc;">' +
+		'<table style="width:100%">' +
+			'<tr style="background-color: #f3f3f3; height:40px;" id="parent">' +
+        '<td style="display: table-cell; color:#007acc" id="target">' +
+					'Cell content</td>' +
+        '</tr>' +
+      '</table></div>';
+		var target = fixture.querySelector('#target'),
+				parent = fixture.querySelector('#parent');
+		var bgNodes = [];
+		var actual = axe.commons.color.getBackgroundColor(target, bgNodes);
+		var expected = new axe.commons.color.Color(243, 243, 243, 1);
+		assert.equal(actual.red, expected.red);
+		assert.equal(actual.green, expected.green);
+		assert.equal(actual.blue, expected.blue);
+		assert.equal(actual.alpha, expected.alpha);
+		assert.deepEqual(bgNodes, [parent]);
+	});
+
+	it('should handle multiple ancestors of the same name', function () {
+		fixture.innerHTML = '<div style="background-color: #007acc;">' +
+		'<table style="width: 100%;">' +
+			'<tr style="background-color: #fff;"><td>' +
+			'<table style="width:100%">' +
+				'<tr style="background-color: #f3f3f3; height:40px;" id="parent">' +
+	        '<td style="display: table-cell; color:#007acc" id="target">' +
+						'Cell content</td>' +
+	        '</tr>' +
+	      '</table>' +
+	    '</td></tr>' +
+	  '</table></div>';
+		var target = fixture.querySelector('#target'),
+				parent = fixture.querySelector('#parent');
+		var bgNodes = [];
+		var actual = axe.commons.color.getBackgroundColor(target, bgNodes);
+		var expected = new axe.commons.color.Color(243, 243, 243, 1);
+		assert.equal(actual.red, expected.red);
+		assert.equal(actual.green, expected.green);
+		assert.equal(actual.blue, expected.blue);
+		assert.equal(actual.alpha, expected.alpha);
+		assert.deepEqual(bgNodes, [parent]);
+	});
+
+	it('should count an implicit label as a background element', function () {
+		fixture.innerHTML = '<label id="target" style="background-color: #fff;">My label' +
```

Comment:
I'd actually remove the background-color value here entirely after learning more about this edge case: if the label is inline and it doesn't have a background-color set on it, the element wouldn't get added to the stack.